src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisInputBuilderService.java

```diff
@@ -141,17 +145,18 @@ private Long getFactorsCount(Map<String, List<UUID>> ids, UUID networkUuid, Stri
         return contAttributesCountTemp;
     }
 
-    private Stream<IdentifiableAttributes> getIdentifiablesFromContainer(SensitivityAnalysisRunContext context, EquipmentsContainer filter,
+    private Stream<IdentifiableAttributes> getIdentifiablesFromContainer(SensitivityAnalysisRunContext context, List<EquipmentsContainer> filters,
         List<IdentifiableType> equipmentsTypesAllowed, Reporter reporter) {
-
-        List<IdentifiableAttributes> listIdentAttributes = goGetIdentifiables(filter, context.getNetworkUuid(), context.getVariantId(), reporter);
+        List<String> containerNamesList = filters.stream().map(EquipmentsContainer::getContainerName).toList();
+        String containerNames = Arrays.toString(containerNamesList.toArray());
+        List<IdentifiableAttributes> listIdentAttributes = goGetIdentifiables(filters, context.getNetworkUuid(), context.getVariantId(), reporter);
```

Comment:
It's better to avoid abbreviation as names
```suggestion
        List<IdentifiableAttributes> listIdentifiableAttributes = goGetIdentifiables(filters, context.getNetworkUuid(), context.getVariantId(), reporter);
```